napari/_qt/dialogs/screenshot_dialog.py

```diff
@@ -41,8 +45,12 @@ def accept(self):
             if os.path.exists(save_path):
                 res = QMessageBox().warning(
                     self,
-                    "Confirm overwrite",
-                    f"{save_path} already exists. Do you want to replace it?",
+                    trans._("Confirm overwrite"),
+                    trans._(
```

Comment:
curious how things like this are supposed to work.  `save_path` obviously never needs to be translated.  Should we (technically) be concatenating the untranslated path with `trans._(" already exists ...")`?

Reply:
curious how things like this are supposed to work, so anything on `{}` is meant to not be translated, but the placement of it might not make sense in other languages other than english (or indoeuropean, which follow very similar structures)

So whenever you have something like

`"{some_variable" + "is not something"`

Then we should always try to do

`"{some_variable} is not something"`

So that some other language can do:

`"is something not {some_variable} "` 

Does that make sense?

Reply:
Ohhh right.  Grammar :)

Reply:
I think tally question was more about the fact that trans/and format are called in the wrong order. 
You want `_("str {value}").format(value=value)` not `_("str {value}".format(value=value))` because of the evaluation order.